.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/wagtailquickcreate
     docker:
-      - image: circleci/python:3.6.4
+      - image: circleci/python:3.8.0
```

Comment:
Looks like the EOF python version is causing pienv can't find a specific version of packages.

Reply:
I agree